packages/@roots/bud-api/src/methods/lazy/index.ts

```diff
@@ -0,0 +1,38 @@
+import {Api} from '@roots/bud-framework'
+
+declare module '@roots/bud-framework' {
+  interface Framework {
+    /**
+     * ## bud.lazy
+     *
+     * Enable experimental lazy compilation. Disabled by default.
+     *
+     * ### Usage
+     *
+     * ```js
+     * bud.lazy()
+     * ```
+     *
+     * Disable:
+     *
+     * ```js
+     * bud.lazy(false)
+     * ```
+     */
+    lazy: Api.Lazy
+  }
+
+  namespace Api {
+    type Lazy = (enabled?: boolean) => Framework
+  }
+}
+
+const lazy: Api.Lazy = function (enabled = true) {
```

Comment:
@QWp6t calling this nets a wild speed improvement. it changes bundles so I don't know if it's a fit for `@roots/sage`. what do you think?